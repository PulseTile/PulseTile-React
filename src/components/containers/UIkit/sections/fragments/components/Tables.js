import React from 'react';

/**
 * This component returns content of Tables section
 */
const Tables = () => {
  return (
    <div id="tables" className="ui-section">
      <strong className="ui-title">Tables</strong>
      <div className="row">
        <div className="col-xs-12 col-sm-6">
          <div className="ui-sub-section">
            <div className="ui-sub-title">Table Simple</div>
            <table className="table">
              <thead>
                <tr>
                  <th className="sorted asc">Cause</th>
                  <th>Reaction</th>
                  <th>Source</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 1</span></td>
                  <td data-th="Reaction"><span>Reaction 1</span></td>
                  <td data-th="Source"><span>Source 1</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 2</span></td>
                  <td data-th="Reaction"><span>Reaction 2</span></td>
                  <td data-th="Source"><span>Source 2</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 3</span></td>
                  <td data-th="Reaction"><span>Reaction 3</span></td>
                  <td data-th="Source"><span>Source 3</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 4</span></td>
                  <td data-th="Reaction"><span>Reaction 4</span></td>
                  <td data-th="Source"><span>Source 4</span></td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
        <div className="col-xs-12 col-sm-6">
          <div className="ui-sub-section">
            <div className="ui-sub-title">Table Striped</div>
            <table className="table table-striped">
              <thead>
                <tr>
                  <th className="sorted asc">Cause</th>
                  <th>Reaction</th>
                  <th>Source</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 1</span></td>
                  <td data-th="Reaction"><span>Reaction 1</span></td>
                  <td data-th="Source"><span>Source 1</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 2</span></td>
                  <td data-th="Reaction"><span>Reaction 2</span></td>
                  <td data-th="Source"><span>Source 2</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 3</span></td>
                  <td data-th="Reaction"><span>Reaction 3</span></td>
                  <td data-th="Source"><span>Source 3</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 4</span></td>
                  <td data-th="Reaction"><span>Reaction 4</span></td>
                  <td data-th="Source"><span>Source 4</span></td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
      <div className="row">
        <div className="col-xs-12 col-sm-6">
          <div className="ui-sub-section">
            <div className="ui-sub-title">Table Hovered</div>
            <table className="table table-hover">
              <thead>
                <tr>
                  <th className="sorted asc">Cause</th>
                  <th>Reaction</th>
                  <th>Source</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 1</span></td>
                  <td data-th="Reaction"><span>Reaction 1</span></td>
                  <td data-th="Source"><span>Source 1</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 2</span></td>
                  <td data-th="Reaction"><span>Reaction 2</span></td>
                  <td data-th="Source"><span>Source 2</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 3</span></td>
                  <td data-th="Reaction"><span>Reaction 3</span></td>
                  <td data-th="Source"><span>Source 3</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 4</span></td>
                  <td data-th="Reaction"><span>Reaction 4</span></td>
                  <td data-th="Source"><span>Source 4</span></td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
        <div className="col-xs-12 col-sm-6">
          <div className="ui-sub-section">
            <div className="ui-sub-title">Table Bordered</div>
            <table className="table table-bordered">
              <thead>
                <tr>
                  <th className="sorted asc">Cause</th>
                  <th>Reaction</th>
                  <th>Source</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 1</span></td>
                  <td data-th="Reaction"><span>Reaction 1</span></td>
                  <td data-th="Source"><span>Source 1</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 2</span></td>
                  <td data-th="Reaction"><span>Reaction 2</span></td>
                  <td data-th="Source"><span>Source 2</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 3</span></td>
                  <td data-th="Reaction"><span>Reaction 3</span></td>
                  <td data-th="Source"><span>Source 3</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 4</span></td>
                  <td data-th="Reaction"><span>Reaction 4</span></td>
                  <td data-th="Source"><span>Source 4</span></td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
      <div className="row">
        <div className="col-xs-12 col-sm-6">
          <div className="ui-sub-section">
            <div className="ui-sub-title">Transform table on mobile devices</div>
            <table className="table table-bordered table-striped rwd-table">
              <thead>
                <tr>
                  <th className="sorted asc">Cause</th>
                  <th>Reaction</th>
                  <th>Source</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 1</span></td>
                  <td data-th="Reaction"><span>Reaction 1</span></td>
                  <td data-th="Source"><span>Source 1</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 2</span></td>
                  <td data-th="Reaction"><span>Reaction 2</span></td>
                  <td data-th="Source"><span>Source 2</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 3</span></td>
                  <td data-th="Reaction"><span>Reaction 3</span></td>
                  <td data-th="Source"><span>Source 3</span></td>
                </tr>
                <tr>
                  <td data-th="Cause" className="sorted asc"><span>Cause 4</span></td>
                  <td data-th="Reaction"><span>Reaction 4</span></td>
                  <td data-th="Source"><span>Source 4</span></td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
        <div className="col-xs-12 col-sm-6">
          <div className="ui-sub-section">
            <div className="ui-sub-title">Table with fixed columns</div>
            <table className="table table-bordered">
              <colgroup>
                <col style={{ width: '26%' }} />
                <col style={{ width: '55%' }} />
                <col style={{ width: '19%' }} />
              </colgroup>
              <thead>
                <tr>
                  <th className="sorted asc">Cause</th>
                  <th>Reaction</th>
                  <th>Source</th>
                </tr>
              </thead>
              <tbody>
               <tr>
                 <td data-th="Cause" className="sorted asc"><span>Cause 1</span></td>
                 <td data-th="Reaction"><span>Reaction 1</span></td>
                 <td data-th="Source"><span>Source 1</span></td>
               </tr>
               <tr>
                 <td data-th="Cause" className="sorted asc"><span>Cause 2</span></td>
                 <td data-th="Reaction"><span>Reaction 2</span></td>
                 <td data-th="Source"><span>Source 2</span></td>
               </tr>
               <tr>
                 <td data-th="Cause" className="sorted asc"><span>Cause 3</span></td>
                 <td data-th="Reaction"><span>Reaction 3</span></td>
                 <td data-th="Source"><span>Source 3</span></td>
               </tr>
               <tr>
                 <td data-th="Cause" className="sorted asc"><span>Cause 4</span></td>
                 <td data-th="Reaction"><span>Reaction 4</span></td>
                 <td data-th="Source"><span>Source 4</span></td>
               </tr>
             </tbody>
           </table>
         </div>
       </div>
     </div>
     <div className="row">
       <div className="col-xs-12 col-sm-6">
         <div className="ui-sub-section">
           <div className="ui-sub-title">Complete table settings</div>
           <table className="table table-striped table-hover table-bordered rwd-table table-sorted">
             <colgroup>
               <col style={{ width: '26%' }} />
               <col style={{ width: '55%' }} />
               <col style={{ width: '19%' }} />
             </colgroup>
             <thead>
               <tr>
                 <th className="sorted asc">Cause</th>
                 <th>Reaction</th>
                 <th>Source</th>
               </tr>
             </thead>
             <tbody>
               <tr>
                 <td data-th="Cause" className="sorted asc"><span>Cause 1</span></td>
                 <td data-th="Reaction"><span>Reaction 1</span></td>
                 <td data-th="Source"><span>Source 1</span></td>
               </tr>
               <tr>
                 <td data-th="Cause" className="sorted asc"><span>Cause 2</span></td>
                 <td data-th="Reaction"><span>Reaction 2</span></td>
                 <td data-th="Source"><span>Source 2</span></td>
               </tr>
               <tr>
                 <td data-th="Cause" className="sorted asc"><span>Cause 3</span></td>
                 <td data-th="Reaction"><span>Reaction 3</span></td>
                 <td data-th="Source"><span>Source 3</span></td>
               </tr>
               <tr>
                 <td data-th="Cause" className="sorted asc"><span>Cause 4</span></td>
                 <td data-th="Reaction"><span>Reaction 4</span></td>
                 <td data-th="Source"><span>Source 4</span></td>
               </tr>
             </tbody>
           </table>
         </div>
       </div>
       <div className="col-xs-12 col-sm-6">
         <div className="ui-sub-section">
           <div className="ui-sub-title">Empty table</div>
             <table className="table table-striped table-hover table-bordered rwd-table table-sorted">
               <colgroup>
                 <col style={{ width: '26%' }} />
                 <col style={{ width: '55%' }} />
                 <col style={{ width: '19%' }} />
               </colgroup>
               <thead>
                 <tr>
                   <th className="sorted asc">Cause</th>
                   <th>Reaction</th>
                   <th>Source</th>
                 </tr>
               </thead>
               <tbody>
                 <tr>
                   <td colspan="3"><span className="label label-default">No items</span></td>
                 </tr>
               </tbody>
             </table>
           </div>
         </div>
       </div>
     </div>
  );
};

export default Tables;
