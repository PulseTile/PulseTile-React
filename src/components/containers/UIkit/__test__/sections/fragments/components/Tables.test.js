import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Tables from '../../../../sections/fragments/components/Tables';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <Tables />', () => {
  it('should renders Tables with props correctly', () => {
    const component = shallow(<Tables />);
    expect(component).toMatchSnapshot();
  });
});
