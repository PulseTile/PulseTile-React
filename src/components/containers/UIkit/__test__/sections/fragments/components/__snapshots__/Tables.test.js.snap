// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Tables /> should renders Tables with props correctly 1`] = `
<div
  className="ui-section"
  id="tables"
>
  <strong
    className="ui-title"
  >
    Tables
  </strong>
  <div
    className="row"
  >
    <div
      className="col-xs-12 col-sm-6"
    >
      <div
        className="ui-sub-section"
      >
        <div
          className="ui-sub-title"
        >
          Table Simple
        </div>
        <table
          className="table"
        >
          <thead>
            <tr>
              <th
                className="sorted asc"
              >
                Cause
              </th>
              <th>
                Reaction
              </th>
              <th>
                Source
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 1
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 1
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 1
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 2
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 2
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 2
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 3
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 3
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 3
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 4
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 4
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 4
                </span>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div
      className="col-xs-12 col-sm-6"
    >
      <div
        className="ui-sub-section"
      >
        <div
          className="ui-sub-title"
        >
          Table Striped
        </div>
        <table
          className="table table-striped"
        >
          <thead>
            <tr>
              <th
                className="sorted asc"
              >
                Cause
              </th>
              <th>
                Reaction
              </th>
              <th>
                Source
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 1
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 1
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 1
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 2
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 2
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 2
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 3
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 3
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 3
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 4
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 4
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 4
                </span>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-xs-12 col-sm-6"
    >
      <div
        className="ui-sub-section"
      >
        <div
          className="ui-sub-title"
        >
          Table Hovered
        </div>
        <table
          className="table table-hover"
        >
          <thead>
            <tr>
              <th
                className="sorted asc"
              >
                Cause
              </th>
              <th>
                Reaction
              </th>
              <th>
                Source
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 1
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 1
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 1
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 2
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 2
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 2
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 3
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 3
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 3
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 4
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 4
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 4
                </span>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div
      className="col-xs-12 col-sm-6"
    >
      <div
        className="ui-sub-section"
      >
        <div
          className="ui-sub-title"
        >
          Table Bordered
        </div>
        <table
          className="table table-bordered"
        >
          <thead>
            <tr>
              <th
                className="sorted asc"
              >
                Cause
              </th>
              <th>
                Reaction
              </th>
              <th>
                Source
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 1
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 1
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 1
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 2
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 2
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 2
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 3
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 3
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 3
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 4
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 4
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 4
                </span>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-xs-12 col-sm-6"
    >
      <div
        className="ui-sub-section"
      >
        <div
          className="ui-sub-title"
        >
          Transform table on mobile devices
        </div>
        <table
          className="table table-bordered table-striped rwd-table"
        >
          <thead>
            <tr>
              <th
                className="sorted asc"
              >
                Cause
              </th>
              <th>
                Reaction
              </th>
              <th>
                Source
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 1
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 1
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 1
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 2
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 2
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 2
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 3
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 3
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 3
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 4
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 4
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 4
                </span>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div
      className="col-xs-12 col-sm-6"
    >
      <div
        className="ui-sub-section"
      >
        <div
          className="ui-sub-title"
        >
          Table with fixed columns
        </div>
        <table
          className="table table-bordered"
        >
          <colgroup>
            <col
              style={
                Object {
                  "width": "26%",
                }
              }
            />
            <col
              style={
                Object {
                  "width": "55%",
                }
              }
            />
            <col
              style={
                Object {
                  "width": "19%",
                }
              }
            />
          </colgroup>
          <thead>
            <tr>
              <th
                className="sorted asc"
              >
                Cause
              </th>
              <th>
                Reaction
              </th>
              <th>
                Source
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 1
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 1
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 1
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 2
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 2
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 2
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 3
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 3
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 3
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 4
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 4
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 4
                </span>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-xs-12 col-sm-6"
    >
      <div
        className="ui-sub-section"
      >
        <div
          className="ui-sub-title"
        >
          Complete table settings
        </div>
        <table
          className="table table-striped table-hover table-bordered rwd-table table-sorted"
        >
          <colgroup>
            <col
              style={
                Object {
                  "width": "26%",
                }
              }
            />
            <col
              style={
                Object {
                  "width": "55%",
                }
              }
            />
            <col
              style={
                Object {
                  "width": "19%",
                }
              }
            />
          </colgroup>
          <thead>
            <tr>
              <th
                className="sorted asc"
              >
                Cause
              </th>
              <th>
                Reaction
              </th>
              <th>
                Source
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 1
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 1
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 1
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 2
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 2
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 2
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 3
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 3
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 3
                </span>
              </td>
            </tr>
            <tr>
              <td
                className="sorted asc"
                data-th="Cause"
              >
                <span>
                  Cause 4
                </span>
              </td>
              <td
                data-th="Reaction"
              >
                <span>
                  Reaction 4
                </span>
              </td>
              <td
                data-th="Source"
              >
                <span>
                  Source 4
                </span>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div
      className="col-xs-12 col-sm-6"
    >
      <div
        className="ui-sub-section"
      >
        <div
          className="ui-sub-title"
        >
          Empty table
        </div>
        <table
          className="table table-striped table-hover table-bordered rwd-table table-sorted"
        >
          <colgroup>
            <col
              style={
                Object {
                  "width": "26%",
                }
              }
            />
            <col
              style={
                Object {
                  "width": "55%",
                }
              }
            />
            <col
              style={
                Object {
                  "width": "19%",
                }
              }
            />
          </colgroup>
          <thead>
            <tr>
              <th
                className="sorted asc"
              >
                Cause
              </th>
              <th>
                Reaction
              </th>
              <th>
                Source
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td
                colspan="3"
              >
                <span
                  className="label label-default"
                >
                  No items
                </span>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
</div>
`;
